README.md

```diff
@@ -51,6 +51,7 @@ LISAv2 includes below test suite categories:
 
 1. Download Web Platform Installer from [here](http://go.microsoft.com/fwlink/p/?linkid=320376&clcid=0x409)
 2. Start Web Platform Installer and select Azure PowerShell (required 6.3.0 or above) and proceed for Azure PowerShell Installation.
+3. Install the Azure Powershell Az module [here](https://docs.microsoft.com/bs-cyrl-ba/powershell/azure/install-az-ps?view=azps-2.5.0)
```

Comment:
I open the URL, seems some words are not English.

Reply:
Fixed to US-en page.